src/test/kotlin/com/pubnub/contract/history/HistorySteps.kt

```diff
@@ -6,7 +6,6 @@ import com.pubnub.contract.state.World
 import io.cucumber.java.en.Then
```

Comment:
In @Then("history response contains messages with {string} and {string} message types")
you could go with like:
  ```
      val messageTypeValuesInAllMessagesInAllChannels =
            fetchMessagesState.pnFetchMessagesResult?.getAllFetchMessageItemsFromAllChannels()
                ?.map { it.messageType.value }

        val uniqueMessageTypeValuesInAllMessagesInAllChannels: HashSet<String> = HashSet(messageTypeValuesInAllMessagesInAllChannels)

        assertThat(
            uniqueMessageTypeValuesInAllMessagesInAllChannels,
            containsInAnyOrder(messageTypeOfFirstMessage, messageTypeOfSecondMessage)
        )
```

Reply:
Yeaaah. Sure. In Kotlin it would be better like that:

```
fetchMessagesState.pnFetchMessagesResult?.getAllFetchMessageItemsFromAllChannels()
              ?.map { it.messageType.value }.distinct()
```

or if you really insist on set than like this:

```
fetchMessagesState.pnFetchMessagesResult?.getAllFetchMessageItemsFromAllChannels()
              ?.map { it.messageType.value }.toSet()
```

by I would prefer the first approach because it's more explicit.